DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/fTSQuerySearchOperators/metadata.json

```diff
@@ -0,0 +1,131 @@
+{
+    "Slug": "fts-query-search-operators",
+    "SourceFileName": "FTSQuerySearchOperators.java",
+    "Title": "FTS Query - Operators",
+    "DescriptionLines": [
+        "**Full Text Search query** allows searching for documents containing particular term(s) within a block of text.",
+        "",
+        "**You can customize the logical operators** between the searched terms.",
+        "",
+        "Upon query, an auto-index is created on the searched document fields."
+    ],
+
+    "Assets": [
+        {
+            "Title": "Operation in queries - Book",
+            "Type": "link",
+            "Url": "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/9-querying-in-ravendb#operations-in-queries"
+        },
+        {
+            "Title": "Full text searching - Book",
+            "Type": "link",
+            "Url": "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/9-querying-in-ravendb#full-text-searching"
+        } /*,
+        {
+            "Title": "How to search",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/latest/java/client-api/session/querying/how-to-use-search"
+        },
+        {
+            "Title": "Querying searching",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/latest/java/indexes/querying/searching"
+        }
+    */
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Define a query",
+            "Slug": "step-1",
+            "DescriptionLines": [
+                "* Define a query on the 'Employees' collection Using the Session `query()`."
+            ],
+            "Assets": [
+                {
+                    "Title": "Query Overview Demo",
+                    "Type": "demo",
+                    "Url": "/java/queries/query-overview"
+                }
+            ]
+        },
+        {
+            "Title": "Define search criteria I",
+            "Slug": "step-2",
+            "DescriptionLines": [
+                "* Use `search()` to search for the specified terms on the specified document field.",
+                "",
+                "* The default operator between 'term1' and 'term2' is OR.<br>",
+                "  Use `SearchOperator.AND` to set the logical AND operation instead.",
+
+                "* In this Search, we search for 'term1' and 'term2' in the document field _Notes_."
```

Comment:
`'term1' and 'term2'` => `_term1 AND term2_`

Reply:
Done